projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/memberships.php

```diff
@@ -262,7 +262,7 @@ public function create_products( $request ) {
 	 *
 	 * @param \WP_REST_Request $request - request passed from WP.
 	 *
-	 * @return WP_Error|array ['products','connected_account_id','connect_url','should_upgrade_to_access_memberships','upgrade_url']
+	 * @return WP_Error|array ['products','connected_account_id','connect_url','upgrade_url']
```

Comment:
I guess `upgrade_url` should maybe be removed as well?

Reply:
Good point. I think we should re-add `should_upgrade_to_access_memberships` since we choose not to remove it from the WPCOM endpoint as yet

Reply:
> I think we should re-add should_upgrade_to_access_memberships since we choose not to remove it from the WPCOM endpoint as yet

We chose not to remove it to make sure we are backward-compatible for users that don't update. But we need to remove it here...

Reply:
Ok, I'll remove it. IMHO, removing it pre-maturely wouldn't accurately represent what the endpoint returns.

Reply:
You need to remove it from WPCOM end anyway. So there will always be a lapse where it won't be accurate, one way or the other.